<!DOCTYPE html>
<meta charset="utf-8">
<!--ref: http://bl.ocks.org/mbostock/1095795-->

<html>
    <link rel="stylesheet" href="main.css" type="text/css">
	<header>
<title> LAMP RFLP </title>
	</header>
	<body>
	    <br/>
	    <h1 align="center">
		     LAMP restriction digest fragment analysis
	    </h1>
	    <div id="list1">
		<ul>
		    <li>Home</li>
		    <li><a href="start.html">Start</a></li>
		</ul>
	    </div>
	    <h2>
		What is it for?
	    </h2>
	    <p>
		This program was created for the analysis of restriction digests of a loop-mediated isothermal amplification (LAMP) reaction <a href="http://en.wikipedia.org/wiki/Loop-mediated_isothermal_amplification">(wiki)</a>. As the products of a LAMP reaction are loop structures which can get sucessively larger as the reaction proceeds <a href="http://loopamp.eiken.co.jp/e/lamp/anim.html">(animation)</a>, using a restriction digest to confirm the products by banding pattern is not as straight-forward as it is with a reaction like polymerase chain reaction (PCR). This program allows for fast analysis by determining the expected banding pattern and outputting it to a virtual gel so that it may be compared to results obtained in a lab setting.<br/><br/> <b>Note: </b>currently this program does not accept loop primers as input, later versions will have this as an option. View the source code on <a href="https://github.com/creisle/creisle.lamprflp">GitHub</a>
	    </p>
	    <h2>
		How it works?
	    </h2>
	    <div id="list2">
		<ol>
		   <li><p><em>Enter fasta input</em>Users must enter their target sequence and primers in fasta format into the first textarea. Tags for primers are restricted to: b1, b2, f1, f2, lb and lf. The target may be tagged whatever name you choose. The program checks user input upon clicking "start reaction" if the primers or target are not enetered correctly, the program will not run. Primers lb and lf as well as the linker sequence are optional but all others are required. </p></li>
		   <li><p><em>Select lane contents</em>Users must select for each lane either a ladder (50 or 100 bp), a retriction enzyme (from the list) or None. If None is selected, the output will be the undigested lamp products. Note that since the undigested products will be loop structures, their migration in the actual gel may be retarded. Therefore it would not be unexpected to observe that the fragments this program predicts are marginally smaller than those found in practice.</p></li>
		   <li><p><em>Start Reaction</em> Upon clicking the "start reaction" button, the program executes with the data currently provided (input may be changed and the button clicked again to re-execute and produce multiple gels). </p></li>
		   <li><p><em>Align Primers</em> The program searches for the best match to f2 and b2 since they are expected to anneal first, using this the f1 and b1 primers are aligned second. Then the overall setup is checked to ensure that the primers in these  positions will admit a lamp reaction. If they will not, the program does not continue</p></li>
		   <li><p><em>Generate Products</em> Produces the LAMP products (As outlined in Notomi et al. (2000). Loop-mediated Isothermal Amplification of DNA.)</p></li>
		   <li><p><em>Digest Products</em> For each selected restriction enzyme, searches the products for cut sites. Stores the fragments (assumes fully digested)</p></li>
		   <li><p><em>Output Image (gel)</em> uses distance travelled = log(bp) to plot the putative pattern of the gel. Note that the blue line can be dragged to use asa navigator in comparing lanes. The value to the right of the blue line is the expected base pairs for a band migrated to that postion</p></li>
		</ol>
	     </div>
         <br/>
         Questions? contact: caralynreisle@gmail.com
	    <br/>
	    <br/>
    </body>
</html>















